packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -79,7 +79,7 @@ class FixedExtentScrollController extends ScrollController {
   /// The returned [Future] resolves when the animation completes.
   ///
   /// The `duration` and `curve` arguments must not be null.
-  Future<Null> animateToItem(int itemIndex, {
+  Future<List<Null>> animateToItem(int itemIndex, {
```

Comment:
The function returns the result of `Future.wait(futures);` which is declared as `Future<List<T>>`, so the return type has to match. Before the execution was throwing an error when finished.